src/config/default.json

```diff
@@ -0,0 +1,4 @@
+{
+  "SERVER_HOST": "localhost",
```

Comment:
Can you make these specific to the Event SDK lib?

Keep in mind that this will be imported as a lib, and the component that imports it may already have configs that match these names. Thus I recommend adding a prefix or parent object to show that the config is applicable to the "EVENT SDK" somehow.

Reply:
see [feature/#803ClientAndServerImpl 12e58d3] Use EVENT_LOGGER_ for config prefix